### PR TITLE
addition of gcs to mongodb template

### PR DESCRIPTION
**Pull request by hhasija**

### CODE REVIEW

File: python/dataproc_templates/gcs/README.md [hunk 1]
```diff
@@ -266,6 +266,81 @@ export JARS=<gcs-bucket-location-containing-jar-file>
...
+                        MongoDB Database Name
+  --gcs.mongo.output.collection GCS.MONGO.OUTPUT.COLLECTION
+                        MongoDB output Collection name
+  --gcs.mongo.batch.size GCS.MONGO.BATCH.SIZE
+                        MongoDB output Batch size
+  --gcs.mongo.output.mode {overwrite,append,ignore,errorifexists}
+                        Output write mode (one of: append,overwrite,ignore,errorifexists) (Defaults to append)                        
+```
+
+## Required JAR files
```
Comment (shashank-google) [inline python/dataproc_templates/gcs/README.md:316]: If possible, please add wget commands to get these jars downloaded from maven repo.

File: python/dataproc_templates/util/template_constants.py [hunk 2]
```diff
@@ -45,6 +45,12 @@
 FORMAT_HBASE = "org.apache.hadoop.hbase.spark"
 TABLE = "table"
 TEMP_GCS_BUCKET="temporaryGcsBucket"
+MONGO_URL = "spark.mongodb.output.uri"
+MONGO_DATABASE = "database"
+MONGO_COLLECTION = "collection"
+FORMAT_MONGO = "com.mongodb.spark.sql.DefaultSource"
+MONGO_DEFAULT_BATCH_SIZE = 512
```
Comment (shashank-google) [inline python/dataproc_templates/util/template_constants.py:52]: why 512 ? Seems very peculiar number.
A round number like 1000 makes more sense to me, unless there is a reason for it.

**Review by shashank-google — APPROVED** [review]
Minor comments, please address.
Please get this review also by either @NiloFreitas @ppaglilla or @surjits254 before merging.

File: python/dataproc_templates/gcs/README.md (hunk 1 above)
Comment (hhasija) [reply python/dataproc_templates/gcs/README.md:316]: Added wget command to download the jar files.

File: python/dataproc_templates/util/template_constants.py (hunk 2 above)
Comment (hhasija) [reply python/dataproc_templates/util/template_constants.py:52]: Actually 512 is the default batch size provided by MongoDB. Hence, I mentioned the same. I have changed it to 1000.

**Review by surjits254 — COMMENTED** [review]
LGTM. will test the code again once Mongo instance is created in GCP

File: python/dataproc_templates/gcs/README.md [hunk 3]
```diff
@@ -266,6 +266,83 @@ export JARS=<gcs-bucket-location-containing-jar-file>
...
+
+```
+sudo wget https://repo1.maven.org/maven2/org/mongodb/spark/mongo-spark-connector_2.12/2.4.0/mongo-spark-connector_2.12-2.4.0.jar
+sudo wget https://repo1.maven.org/maven2/org/mongodb/mongo-java-driver/3.9.1/mongo-java-driver-3.9.1.jar
+```
+
+## Example submission
+
+```
+export JARS=<gcs-bucket-location-containing-jar-file>
```
Comment (surjits254) [inline python/dataproc_templates/gcs/README.md:332]: can we add other export commands in the readme?

**Review by surjits254 — CHANGES_REQUESTED** [review]

File: python/dataproc_templates/gcs/README.md (hunk 3 above)
Comment (hhasija) [reply python/dataproc_templates/gcs/README.md:335]: Added other export commands as well.

**Review by surjits254 — APPROVED** [review]

**Pull request merged**